src/main/java/com/minecraftabnormals/upgrade_aquatic/common/entities/jellyfish/helper/JellyfishSizeMap.java

```diff
@@ -0,0 +1,117 @@
+package com.minecraftabnormals.upgrade_aquatic.common.entities.jellyfish.helper;
+
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.Map;
+import java.util.Random;
+import java.util.Set;
+import java.util.TreeMap;
+
+public class JellyfishSizeMap {
```

Comment:
Many of the local instance field accesses in this class don't use `this`.
There are also many methods in this class that I don't see being useful in the future and some that should be annotated with `@Nullable`.

Reply:
I've removed the comparator and the floor and ceiling methods, as well as adding `@Nullable` to `get` and the higher/lower methods. Is there anything else that's redundant?

Reply:
Seems good to me now.